### PR TITLE
AT_05.04.03 | <New item> User is able to create Folder project

### PR DESCRIPTION
**Pull request by Olgalex**
https://trello.com/c/zZ1QWc5k

### CODE REVIEW

**Comment by NatalyaVasylyeva** [comment]
Hi!
You should add TC card to your AT card in Trello.
"http://localhost:8080/view/all/newJob" your code is hard coded. 
You should type the full name of your AT card to Pull Request.
Your AT card in Trello should have an understandable title. <New item> Folder - is not appropriate. The title should display your validation/check. I can't understand in the title what check is in your test case.

**Comment by Olgalex** [comment]
Thank you, @NatalyaVasylyeva

File: cypress/e2e/createJob.cy.js [hunk 1]
```diff
@@ -9,4 +9,14 @@ describe('AT_02.01_002 | Homepage Create a clickable job link', () => {
...
     });
+    it ('AT_05.04.03 | <New item> User is able to create Folder project', () => {
+        cy.get("a[href='/view/all/newJob']").should('have.text', 'New Item').click()
+        cy.url().should("eq", 'http://localhost:8080/view/all/newJob')
+        cy.get('#name').type('olgalex').should('be.visible')
+        cy.get('.label').contains('Folder').click();
+        cy.get('#ok-button').click()
+        cy.get('#general').should('be.visible')
+        cy.get("button[name='Submit']").click()
+        cy.get("div[id='main-panel'] h1").should('have.text', '\n    olgalex\n  ')
```
Comment (DariaMas7) [inline cypress/e2e/createJob.cy.js:20]: \n    olgalex\n   should be in fixtures

**Review by DariaMas7 — COMMENTED** [review]
Hello could you please move all text to fixtures

**Review by ElNov686 — APPROVED** [review]

**Review by DariaMas7 — APPROVED** [review]

**Pull request merged**